Source/Core/Logging/FramesEventLogger.swift

```diff
@@ -37,6 +37,21 @@ final class FramesEventLogger: FramesEventLogging {
       self.dateProvider = dateProvider
     }
 
+    static func buildRemoteProcessorMetadata(environment: Environment,
+                                             appPackageName: String,
+                                             appPackageVersion: String,
+                                             uiDevice: UIDevice) -> RemoteProcessorMetadata {
+
+            return RemoteProcessorMetadata(productIdentifier: Constants.productName,
+                                           productVersion: Constants.version,
+                                           environment: environment.rawValue,
+                                           appPackageName: appPackageName,
+                                           appPackageVersion: appPackageVersion,
+                                           deviceName: uiDevice.modelName,
+                                           platform: "iOS",
```

Comment:
```suggestion
                                           platform:  platform,
```

Reply:
sorry wht do you mean by this?

Reply:
I suggest to use native device OS parameter. we may use payment for other OSs like WatchOS .

Reply:
I suggest to use native device OS parameter. we may use payment for other OSs like WatchOS .